FirebaseAuthUI/FUIPasswordSignUpViewController.h

```diff
@@ -26,9 +28,9 @@ NS_ASSUME_NONNULL_BEGIN
 @interface FUIPasswordSignUpViewController : FUIAuthBaseViewController
 
 /** @property footerTextView
-    @brief The text view in the footer of the table.
+    @brief The text view in the footer of the table that displays Privacy and Terms of Service.
```

Comment:
nit: it's no longer accurate to describe this view as a text view.

Reply:
Done.